.github/workflows/check-providers.yml

```diff
@@ -111,7 +111,7 @@ jobs:
       - name: >
           Remove incompatible Python ${{ matrix.python-version }} provider packages
         run: |
-          echo "Removing Python 3.8-incompatible provider: cloudant"
+          echo "Removing Python 3.9-incompatible provider: cloudant"
```

Comment:
There is a bug two lines above - `${{ matrix.python-version }}` is used - should be 3.9 explicitly.

Reply:
Adjusted.